src/state/slices/txHistorySlice/txHistorySlice.ts

```diff
@@ -233,8 +176,9 @@ export const txHistory = createSlice({
     clear: () => {
       return initialState
     },
-    onMessage: (txState, { payload }: TxMessage) =>
-      updateOrInsertTx(txState, payload.message, payload.accountId),
+    onMessage: (txState, { payload }: TxMessage) => {
+      updateOrInsertTxs(txState, [payload.message], payload.accountId)
```

Comment:
Urgh, yeah that was most definitely wrong!

Reply:
I mean they were pretty much dupes doing the loop slightly different, no need to maintain both and complicate things haha